src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -740,17 +744,33 @@ namespace SystemAvailabilityManager {
 					ErrorsFound = true;
 				}}
 
+				// Cycling Run Time Control Type
+				if ( !lAlphaFieldBlanks( 5 ) ) {
+					{ auto const SELECT_CASE_var( MakeUPPERCase( cAlphaArgs( 5 ) ) );
```

Comment:
In the future, try to avoid propagating the `SELECT_CASE_var` code.  It was really just used when the Fortran code was changed over but lingered.  Nbd for here, just don't add any others in the future.  :+1: